README.md

```diff
@@ -48,298 +48,400 @@ The FWS Payments contract enables ERC20 token payment flows through "rails" - au
 
 - **Client —** An account that *pays* a service provider (also referred to as the *payer*)
-- **Service Provider** — An account managed the provider of some service which receives payment from a client (also referred to as the *payee*).
+- **Service Provider** — An account managed by the provider of some service which receives payment from a client (also referred to as the *payee*).
 
 ### Rail
 
 A rail along which payments flow from a client to a service provider. Rails track lockup, maximum payment rates, and obligations between a client and a servive provider. Client ↔ Service Provider pairs can have multiple payment rails between them but they can also reuse the same rail across multiple deals. Importantly, rails:
 - Specify the maximum rate at which the client will pay the service provider, the actual amount paid for any given period is subject to validation by the **validator** described below.
-- Specify the period in advance the client is required to lock funds (the **lockup period**). There’s no way to force clients to lock funds in advance, but we can prevent them from *withdrawing* them and make it easy for service providers to tell if their clients haven’t met their lockup minimums, giving them time to settle their accounts.
+- Define the required funding commitment for future payments through the **lockup period**. While the contract cannot force a client to deposit funds from their external wallet, it strictly enforces lockups on all funds held within their contract account. It prevents clients from withdrawing locked funds and blocks operator actions that would increase a client's lockup obligation beyond their available balance. This system provides a reliable way for service providers to verify a client's funding commitment for the agreed-upon period.
 
 ### Validator
 
-An validator is an (optional) smart contract that can validate / arbitrate payments associated with a single rail. For example, a payment rail used for PDP will specify the PDP service as its validator. A validator can:
+A validator is an optional contract that acts as a trusted arbitrator for a rail. Its primary role is to validate payments during settlement, but it also plays a crucial part in the rail's lifecycle, especially during termination.
 
-- Prevent settlement of a payment rail entirely.
-- Refuse to settle a payment rail past some epoch.
-- Reduce the amount paid out by a rail for a period of time (e.g., to account for actual services rendered, penalties, etc.).
+When a validator is assigned to a rail, it gains the ability to:
+
+-   **Mediate Payments:** During settlement, a validator can prevent a payment, refuse to settle past a certain epoch, or reduce the payout amount to account for actual services rendered, penalties, etc.
```

Comment:
nit: I've found the deleted bullet list useful.  Maybe restructure them to sub-bullets here.

Reply:
@ZenGround0 Not sure what you mean, The new bullet list has everything the old list has but in more detail